### PR TITLE
Fix pvc alert teardown

### PR DESCRIPTION
**Pull request by jgarciao**
- Add `Delete Folder In User Notebook` keyword
- Fix pvc alert teardown
- Move PVC alert tests from Sanity to Tier2

### CODE REVIEW

**Comment by jgarciao** [comment]
Tested with Tier2 pipeline successfully here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-tier2/1

File: tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot [hunk 1]
```diff
@@ -155,20 +155,43 @@ Clean Up User Notebook
...
+  ${oc_whoami} =  Run   oc whoami
+  IF    '${oc_whoami}' == '${admin_username}'
+      # We import the library here so it's loaded only when we are connected to the cluster
+      # Having the usual "Library OpenShiftCLI" in the header raises an error when loading the file
+      # if there is not any connection opened
+      Import Library    OpenShiftCLI
+
+      # Verify that the jupyter notebook pod is running
+      ${notebook_pod_name} =   Get User Notebook Pod Name  ${username}
+      Search Pods    ${notebook_pod_name}  namespace=rhods-notebooks
```
Comment (lugi0) [inline tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: Isn't this mostly a repetition of `Clean Up User Notebook`? Did you mean to remove the other keyword?

Comment (jgarciao) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: This one removes an specific folder that you pass as a parameter. I think both can be useful

Comment (lugi0) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: Can't they be merged into one keyword? perhaps by setting the default value for `${folder}` to empty

Comment (jgarciao) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: Personally I think it's more clear like this, one is to clear everything (except hidden files)  and the other to delete an specific folder. Maybe the fist one could have a better name (Empty User Notebook?)

`Clean Up User Notebook  ${OCP_ADMIN_USER.USERNAME}  ${TEST_USER.USERNAME}`

`Delete Folder In User Notebook  ${OCP_ADMIN_USER.USERNAME}  ${TEST_USER.USERNAME}  foldername`

Comment (lugi0) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: I personally prefer to keep code repetition to a minimum, but at this point it's just a question of personal preference. I think it works like this as well, I just wanted to check if you *really* wanted to have those two separate keyword or not.

Apart from this, lgtm

Comment (vasukulkarni) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:192]: resolving for now, but we can discuss in our call later.

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**